src/tests/curve.rs

```diff
@@ -355,6 +356,11 @@ pub fn hash_to_curve_test<G: CurveExt>() {
     }
 }
 
+pub fn endo_consistency_test<G: CurveExt>() {
+    let g = G::generator();
```

Comment:
Can we use a random value here? To maximize chances of catching an error??
I'd also try to use the fn in a loop 1000 times or similar. Which executes fast but increases chances to find errors.

Reply:
Done https://github.com/privacy-scaling-explorations/halo2curves/pull/129/files#diff-e654b43be305459ee924b3b7b2cf6142dab1f359ceec19903abe85f591b10df8R426-R438